WalletWasabi.Fluent/ViewModels/AddWallet/WalletNamePageViewModel.cs

```diff
@@ -50,8 +50,18 @@ private async Task OnNextAsync()
 		switch (options)
 		{
 			case WalletCreationOptions.AddNewWallet add:
-				Navigate().To().RecoveryWords(add);
-				break;
+				var dialogCaption = "This is needed to open and to recover your wallet.\n" +
+				                    "It will be used as the 13th seed word, it can never be changed.";
```

Comment:
"It will be used as the 13th seed word, it can never be changed."
=>
This will add passphrase-protection for your private key, it can never be changed.

@yahiheb @Kruwed WDYT?

Reply:
I much prefer "It will be used as the 13th seed word, it can never be changed."

If I were to improve it, I use the word "reset" instead of the word "changed" or in addition to it.